src/SFE.scss

```diff
@@ -55,3 +55,9 @@ button {
 input[type="search"] {
   appearance: none;
 }
+
+// Paragon-specific styling copied here from:
+// https://github.com/edx/paragon/blob/efd86c59fe6266b569939a2d5f526fce5f2ecafc/src/asInput/asInput.scss#L5-L7
+.fa-icon-spacing {
```

Comment:
Like I explained in-person, I think we should start using the CSS generated by Paragon's themeable build so we don't have to keep copy-and-pasting CSS from Paragon. It includes Bootstrap and Font Awesome, so we will have to do some refactoring to use those from Paragon instead of importing it separately in studio-frontend. In the interest of getting this merged soon, I created a separate ticket to do that work: https://openedx.atlassian.net/browse/EDUCATOR-2704